compiler/src/dotty/tools/io/ZipArchive.scala

```diff
@@ -112,9 +112,15 @@ abstract class ZipArchive(override val jpath: JPath) extends AbstractFile with E
   def close(): Unit
 }
 /** ''Note:  This library is considered experimental and should not be used unless you know what you are doing.'' */
-final class FileZipArchive(jpath: JPath) extends ZipArchive(jpath) {
+final class FileZipArchive(jpath: JPath, release: Option[String] = None) extends ZipArchive(jpath, release) {
```

Comment:
I suggest removing the default param so we don' forget to pass release in some situations:
```suggestion
final class FileZipArchive(jpath: JPath, release: Option[String]) extends ZipArchive(jpath, release) {
```